### DISCUSSION TITLE
Crash condition openvaf 23.5.0

### DISCUSSION THREAD

**Discussion by holmesozic** [pascalkuthe/OpenVAF#144] [General]
Reporting a crash condition that model without parameters crashes compiler.

[openvaf-crash-1724436646.log](https://github.com/user-attachments/files/16732462/openvaf-crash-1724436646.log)

`include "discipline.h"
`include "constants.h"
module branchtest(a,b);
	inout         a,b;
	electrical    a,b;
	// uncomment unused parameter and compiler passes
	// parameter real c=1.0;
	branch (a,b) ab;
	analog begin
		I(ab) <+ V(ab);
	end
endmodule

**Comment by gjcoram** [comment]
This is a known issue: https://github.com/pascalkuthe/OpenVAF/issues/103
but is fixed in this commit: https://github.com/gjcoram/OpenVAF/commit/fdf04734f0bd58024588b5aa28a82cc0a72d9aee
